nvtabular/inference/triton/model_hugectr.py

```diff
@@ -86,9 +86,7 @@ def execute(self, requests: List[InferenceRequest]) -> List[InferenceResponse]:
                 output_tensors.append(Tensor("DES", np.array([[]], np.float32)))
 
             if "cats" in self.column_types:
-                output_df[self.column_types["cats"]] = (
-                    output_df[self.column_types["cats"]] + self.slot_sizes
-                )
+                output_df[self.column_types["cats"]] += self.offsets
                 cats_np = _convert_cudf2numpy(output_df[self.column_types["cats"]], np.int64)
                 output_tensors.append(
```

Comment:
`row_index = np.arange(len_cats_np + 1, dtype=np.int32).reshape(1, len_cats_np + 1)`
The current csr conversion supported by this line is only for the case where each slot contains only one cat feature, right?
In other words, the current script only supports the conversion of one-hot encoded categorical data. The number of cat features is equal to the number of slots (cats_np.shape[1] = slot_sizes.size)

Reply:
Yes, we only support one-hot encoding right now.